### PR TITLE
Replace proxylocal with ngrok

### PR DESCRIPTION
**Pull request by michaelwmcnamara**
This is a small update to the docs on testing locally replace the tips on how to use proxylocal with information on how to use ngrok.
This is because proxylocal is no longer being updated.
See https://github.com/proxylocal/proxylocal-gem/issues/16

### CODE REVIEW

File: docs/03-dev-howtos/09-testing-externally-on-localhost.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
-# Testing externally on your localhost 
+# Testing externally on your localhost
 ====================================
 
+There are two options for this. Proxylocal and ngrok. Proxy local has the advantage of being able to set a
+constant url for your personal use, however ngrok is useful for
+
```
Comment (katebee) [inline docs/03-dev-howtos/09-testing-externally-on-localhost.md]: for....

??? Don't leave me in suspense!

**Comment by katebee** [comment]
I know line 6 is not part of your changes, but is it correct? `./sbt001` is not recognised as a command

Should it be `./sbt`?

File: docs/03-dev-howtos/09-testing-externally-on-localhost.md (hunk 1 above)
Comment (michaelwmcnamara) [inline docs/03-dev-howtos/09-testing-externally-on-localhost.md]: Ah - very good spot! Thanks!

**Comment by michaelwmcnamara** [comment]
Agree - I've never heard of ./sbt001. I've removed it from the doc.

**Comment by michaelwmcnamara** [comment]
Updating change to remove all reference to proxylocal (I've updated the title & description to reflect this).

**Comment by NataliaLKB** [comment]
Small comment, but otherwise great! 👍

File: docs/03-dev-howtos/09-testing-externally-on-localhost.md [hunk 2]
```diff
@@ -1,37 +1,24 @@
...
 ```bash
-proxylocal 9001
-```
-Edit your article conf (ie. _article/conf/env/DEV.properties_) with the name of the proxylocal server you've been assigned:
-```
-guardian.page.coreNavigationUrl=http://kj56.t.proxylocal.com
+brew cask install ngrok
 ```
-Then run your article server on some other port:
+In sbt run your article server or dev-build server:
```
Comment (michaelwmcnamara) [inline docs/03-dev-howtos/09-testing-externally-on-localhost.md]: Great suggestion! I have updated this accordingly :-)

**Pull request merged**